src/doc_scan_service/session/create/session.specification.js

```diff
@@ -35,6 +37,7 @@ class SessionSpecification {
    */
   constructor(
     clientSessionTokenTtl,
+    sessionDeadline,
```

Comment:
It seems logical to have it as second parameter, but if we were to consider anyone using this constructor directly, that would be a breaking change.
Any reason (beyond the grouping by theme) to have it in second position, was it like that in the Java SDK ?

Reply:
Ah yes, you're right. I didn't consider that. The reason for putting it there was just logical grouping, but if it's going to be breaking then I don't think it's worth it

Reply:
Yeah, let's change it in case that is introducing a breaking change for some users (despite they should be using the builder).